requirements/base.in

```diff
@@ -12,4 +12,4 @@ pythreejs
 sciline>=24.06.0
 scipp>=23.8.0
 scippneutron>=23.9.0
-scippnexus>=23.12.1
+scippnexus>=24.6.0
```

Comment:
Since ScippNexus works also with lower versions, I also do not see a reason for the lower bound here. Revert to 23.12.1 and let `pip-compile-multi` deal with the rest?